src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -455,6 +467,38 @@ private void getCanvasResponse(HttpServletRequest request,
 		}else {
 			url=canvasURL+pathInfo;
 		}
+
+		TcSessionData tc = (TcSessionData) request.getSession().getAttribute(TC_SESSION_DATA);
+		M_log.debug("TC Session Data: " + tc);
+
+		//useful for debugging
```

Comment:
How about making these two sections into methods and then just not calling them unless you need them for debugging?  Otherwise you'll get lots of "take out that commented code" comments.